src/alfasim_sdk/_internal/alfacase/case_description_attributes.py

```diff
@@ -1,3 +1,4 @@
+import textwrap
 from enum import EnumMeta
```

Comment:
Don't skip this file.

Reply:
> ?

There are a lot of generated code around and some of the files I did edit manually require reviewers to click "Load diff" because they are not being rendered by default.
Because of that I added those. I think I could have written better.

Reply:
Ahh OK.